serial/serbridge.c

```diff
@@ -463,9 +464,17 @@ serbridgeInitPins()
   // set both pins to 1 before turning them on so we don't cause a reset
   if (mcu_isp_pin >= 0)   GPIO_OUTPUT_SET(mcu_isp_pin, 1);
   if (mcu_reset_pin >= 0) GPIO_OUTPUT_SET(mcu_reset_pin, 1);
+  // set TX_ENABLE to 0 so we start up listening
+  if (mcu_reset_pin >= 0) GPIO_OUTPUT_SET(uart0_tx_enable_pin, 0);
```

Comment:
`if (uart0_tx_enable_pin >= 0) GPIO_OUTPUT_SET(uart0_tx_enable_pin, 0);`
Start listening even reset pin is disable.